save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/mobile/AboutUsMobileView.kt

```diff
@@ -26,6 +29,7 @@ import web.cssom.rem
 @OptIn(ExperimentalJsExport::class)
 class AboutUsMobileView : AboutUsView() {
     override fun ChildrenBuilder.render() {
+        particles()
```

Comment:
Does it work after all?

Reply:
yep, the only place where it does not work - is VFC

Reply:
That is actually still a problem as class components seem to be deprecated. Need to fix it somehow later

Reply:
> That is actually still a problem as class components seem to be deprecated. Need to fix it somehow later
yeah, very old and deprecated, but still got a lot of problems with VFC

Reply:
What was the problem?